ozaria/engine/cinematic/CinematicLankBoss.js

```diff
@@ -85,7 +85,8 @@ export default class CinematicLankBoss {
     }
 
     const lHero = getLeftHero(shot)
-    const original = (me.get('heroConfig') || {}).thangType || HERO_THANG_ID
+    // TODO: Replace hard coded hero with user hero
+    const original = getHeroSlug()
```

Comment:
Suggestion: `getHeroSlugConstant() || (me.get('heroConfig') || {}).thangType || HERO_THANG_ID`
Also, is this only for testing purposes? Are we keeping track somewhere that we need to update this later?

Reply:
We are using the hero constant for content creation. There is an asana task to replace this behavior with the users chosen hero once we have that system. I am not sure where we have captured the work to choose the hero.

Reply:
And we're doing this from a code side because the content is written and it would be a lot of effort to modify the content to point to the hero slug and then change it back to the hero.